Experiments/Experiments/DefaultFeatureFlagService.swift

```diff
@@ -47,6 +47,8 @@ public struct DefaultFeatureFlagService: FeatureFlagService {
             return buildConfig == .localDeveloper || buildConfig == .alpha
         case .inbox:
             return buildConfig == .localDeveloper || buildConfig == .alpha
+        case .bulkEditProductVariations:
+            return buildConfig == .localDeveloper
```

Comment:
Its a common practice to add the `|| buildConfig == .alpha` so testes can test help out while we develop features.

```swift
/// Production-like build but with more enabled to help people test branches
/// that might be behind feature flags.
case alpha
 ```

Reply:
Fixed 👍